doc/additional_linux_details.rst

```diff
@@ -184,10 +192,39 @@ connection to the device, and then use ``snmpwalk``::
    snmpget -v1 -c public 192.168.0.50 1.3.6.1.2.1.1.4.0
    snmpset -v1 -c private 192.168.0.50 1.3.6.1.2.1.1.4.0 s "My new sys contact"
 
+If you enable debug logging in the p-net stack, the two last commands will
+cause entries in the p-net log.
+
 See :ref:`network-topology-detection` for more details on SNMP.
 
 
 snmpd in a Yocto build
 ----------------------
 In an embedded Linux Yocto build, you would include the ``snmpd`` daemon by
 using the ``net-snmp`` recipe.
+
+
+Persistent logs
+---------------
+To make the journalctl logs persistent between restarts::
+
+   sudo mkdir -p /var/log/journal
+   sudo systemd-tmpfiles --create --prefix /var/log/journal
+
+Remove all contents of the journalctl logs::
+
+   sudo journalctl --rotate
+   sudo journalctl --vacuum-time=1s
+
+
+Boot time optimization
+----------------------
+The boot time should be less than approximately 15 seconds, for the "missing
+peer" alarm to be sent within 30 s after the power on.
+
+To improve the startup time of your Linux device, it is useful to study what
+is delaying the start. Use these commands to analyze the startup::
```

Comment:
This obviously only works if you are using systemd.

Reply:
```
If you use the "systemd" init system, you can use these
commands to analyze the startup::
```
Done